tests/unit/mathematics/linalg/LinalgBackendViennaCL_unittest.cc

```diff
@@ -5,7 +5,7 @@
 #include <thread>
 
 #ifdef HAVE_VIENNACL
-#include <shogun/mathematics/linalg/LinalgBackendViennaCL.h>
+#include <shogun/mathematics/linalg/LinalgBackendViennacl.h>
```

Comment:
@lambday Sorry I believe it's a bug..It doesn't distinguish cases when I run the codes at local so I didn't notice..